Add a note in v3.0 pointing to a later doc version (#7533)
pingcap/docs: query-execution-plan.md

Based on the details of your tables, the TiDB optimizer chooses the most efficient query execution plan, which consists of a series of operators. This document details the execution plan information returned by the `EXPLAIN` statement in TiDB.

> **Note:**
>

> > You are reading an older version of Understand the Query Execution Plan (TiDB v3.0). [`EXPLAIN` Overview](https://docs.pingcap.com/tidb/stable/explain-overview) is the newer version of this document. It is recommended to read the newer version if do not have any special need.

Reviewer: ```suggestion
> You are reading an earlier version of Understand the Query Execution Plan (TiDB v3.0). [`EXPLAIN` Overview](https://docs.pingcap.com/tidb/stable/explain-overview) is the latest version of this document and is recommended if you do not have any special needs.
```
